tests/test_nonbonded.py

```diff
@@ -445,9 +445,9 @@ def test_nonbonded(self):
             for cutoff in [1.0]:
                 # E = 0 # DEBUG!
                 charge_params, ref_potential, test_potential = prepare_water_system(
-                    subset, lambda_plane_idxs, lambda_offset_idxs, p_scale=1.0, cutoff=cutoff
+                    subset, lambda_plane_idxs, lambda_offset_idxs, p_scale=5.0, cutoff=cutoff
                 )
-                for precision, rtol, atol in [(np.float64, 1e-8, 1e-8), (np.float32, 1e-4, 3e-5)]:
+                for precision, rtol, atol in [(np.float64, 1e-8, 1e-8), (np.float32, 1e-4, 5e-4)]:
```

Comment:
By changing the box size, the precision went down. This precision is in line with other float32 tests

Reply:
Do you mind dumping out the raw forces? I wonder if the small water box is causing clashes at the boundary.